dist/src/main/config/gateway.yaml.template

```diff
@@ -190,6 +190,21 @@
       # Sets the number of threads the gateway will use to communicate with the broker cluster
       # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_THREADS_MANAGEMENTTHREADS.
       # managementThreads: 1
+      #
```

Comment:
:wrench: This should be also added in `broker.yaml` for embedded gateway.

Reply:
I thought about it, but if you look at it, we put none of the embedded gateway settings there. Only `enabled`. Or am I missing something? :thinking: